projects/layout/components/cell/cell.options.ts

```diff
@@ -0,0 +1,12 @@
+import {tuiCreateOptions} from '@taiga-ui/cdk/utils/di';
+import type {TuiSizeL, TuiSizeS} from '@taiga-ui/core/types';
+
+export interface TuiCellOptions {
+    readonly height: 'compact' | 'normal' | 'spacious';
+    readonly size: TuiSizeL | TuiSizeS | '';
+}
+
+export const TUI_CELL_DEFAULT_OPTIONS: TuiCellOptions = {height: 'normal', size: 'l'};
```

Comment:
Let's inline this object in the method call below

Reply:
fixed